sig-network/annual-report-2024.md

```diff
@@ -94,9 +94,30 @@ or implementation support is the key to "unblocking" them:
 
 3. Did you have community-wide updates in 2024 (e.g. KubeCon talks)?
 
-<!--
-  Examples include links to email, slides, or recordings.
--->
+Blog Posts:
+
+- [Getting started with the AdminNetworkPolicy](https://network-policy-api.sigs.k8s.io/blog/2024/01/30/getting-started-with-the-adminnetworkpolicy-api/)
+- [Gateway API v1.1](https://kubernetes.io/blog/2024/05/09/gateway-api-v1-1/)
+- [Gateway API v1.2](https://kubernetes.io/blog/2024/11/21/gateway-api-v1-2/)
+
+Talks:
+
+- [Network Policy: The future of Network Policy (Kubecon EU)](https://sched.co/1Yhhb)
+- [How many network policies can I create? (Kubecon EU)](https://kccnceu2024.sched.com/event/1YeMI)
+- [Gateway API "Beyond GA" (Kubecon EU)](https://www.youtube.com/watch?v=LITg6TvctjM)
```

Comment:
Should probably be consistent about sched vs youtube links?
Or have both for all of them (since you can usually get the slides at the sched link).

Reply:
Yeah, fixed :+1: